### ISSUE TITLE
Fritz Box Setting rename "Allow access for applications"

### ISSUE THREAD

**Issue by jannemannX** [home-assistant/home-assistant.github.io#35787]
### Feedback

The fritzbox setting is now named "Allow access for applications" instead of "Permit access for apps". In the "Important" section.

### URL

https://www.home-assistant.io/integrations/fritz/

### Version

2024.11.2

**Comment by mib1185** [comment]
Hi @jannemannX based on your report, i assume you refer to an FirtzOS update? if yes, on which FritzOS version did you upgrade?

**Comment by jannemannX** [comment]
I'm on 7.59, actually not sure when I updated last. I just assumed they changed the wording of the setting and its not yet reflected in the documentation. 

See here.
![image](https://github.com/user-attachments/assets/98ee6c24-257b-4e6a-adf4-cd21ada075e7)

**Comment by mib1185** [comment]
I've recently updated the docs based on my 7530 ax on FritzOS 7.81 (_#35589_) and it shows this:

![image](https://github.com/user-attachments/assets/1fd52b4f-c751-4769-b948-d315dbc37cf1)

**Comment by jannemannX** [comment]
Oh interesting, my bad then.
This is the most up to date version for my fritzbox 7583 however so probably just add both to the docs?

**Comment by mib1185** [comment]
I don't think we should should add every possible variation of an options description (_where to start and where to end?_). At most a hint about "option names and descriptions may vary between different FritzOS versions" or similar

**Comment by jannemannX** [comment]
Yeah sure, wouldnt have made the ticket if I had known that there are different up to date versions of fritzos with varying names for settings :D
Also i found the newer name kind of odd
